src/program/lwaftr/tests/config/test_env.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+function random_name {
+    cat /dev/urandom | tr -dc 'a-z' | fold -w 20 | head -n 1
+}
+
+# TEST_DIR is set by the caller.
+DATA_DIR="${TEST_DIR}/data"
+BENCHDATA_DIR="${TEST_DIR}/benchdata"
+
+# Start the lwaftr process. The process should end when the script ends;
+# however, if something goes wrong and it doesn't end correctly,
+# a duration is set to prevent it running indefinitely.
+function start_lwaftr_bench {
+    ./snabb lwaftr bench --reconfigurable --bench-file /dev/null --name "$1" \
+	                       --duration 30 \
+                         program/lwaftr/tests/data/icmp_on_fail.conf \
+                         program/lwaftr/tests/benchdata/ipv{4,6}-0550.pcap &> /dev/null &
+
+    # Not ideal, but it takes a little time for the lwaftr to properly start.
+    sleep 2
+}
```

Comment:
This could be made consistent with how the "lwaftr query" test handles multiple processes which need to run in the background by using tmux sessions. This method works and it would require re-writing and since this works it's definitely not a requirement for my  LGTM - just something if you feel so inclined.